data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Bring Hai: done"
+
+
+
+mission "Unfettered: Bring Medical Supplies"
+	landing
+	name `Bring medicals to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	source "Darkcloak"
+	stopover "Snowfeather"
+	cargo "medicals" 50
+	blocked `You need <tons> of cargo space available to carry the medicals supplies.`
+	deadline 8
+	to offer
+		has "Unfettered: Bring Hai: done"
+		has "reputation: Hai" >= 0
+	substitutions
+		<prioritymessage> "Unfortunately, because it's in Hai, you can only guess at the meaning out of it, but it is clearly important. You can see the Hai word for "Unfettered" repeated several times, as well as numbers in the hundreds."
+			not "language: Wanderer"
+		<prioritymessage> `"Assistance requested from all available pilots. Local outbreak of broiling sickness has spread catastrophically in Unfettered space, and is now classified as an uncontrolled [epidemic, plague]. Hundreds are already critically ill and Unfettered hospitals have declared an emergency situation, fearing the consequences of current [medical, treatment] supply levels. Medical supplies are to be retrieved from <stopovers> for immediate return."`
+			has "language: Wanderer"
+	on offer
+		conversation
+			`When you land, there is a message repeating on priority channels in Hai standard and tagged with the information for one of the hospitals nearby. <prioritymessage> Unsure whether they would accept your assistance as an outsider, especially given the message is in Hai, you decide to first look for more information planet-side. After all, it is unlikely that a Hai disease would be of any risk to you.`
+			`	When landing on an Unfettered planet, even after having brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered rush around, and various vehicles are frequently coming and going, bringing both supplies and the infected to the hospital. The hospital itself looks different than the rest of the Unfettered structures: it seems more organized, proper. It is probably the best maintained structure here alongside the shipyards - or at least it has that appearance. One part of it looks to have been grafted onto the rest though, and seems to be a recent addition. Not even their critical buildings appear to be totally safe from the Unfettered's propensity for haphazard building extensions.`
+			`	Many pilots seem to have taken up the call, but it appears the organizers here are still interested in all the volunteers they can get. Now that you're here, you're not sure you'll be able to leave without at least being accosted for your assistance with this.`
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+					goto agree
```

Comment:
If you don't have the language to read the message this choice makes no sense at this point.

This mission offering probably needs a whole branch to it just for investigating and finding out what's going on if you don't have the ability to interpret Hai.

Reply:
done, I added a thing where you hail a nearby unfettered ship and they somewhat translate the gist of it